router/core/websocket.go

```diff
@@ -110,14 +120,6 @@ func NewWebsocketMiddleware(ctx context.Context, opts WebsocketMiddlewareOptions
 			handler.forwardQueryParamsConfig.withStaticAllowList = len(handler.forwardQueryParamsConfig.staticAllowList) > 0
 			handler.forwardQueryParamsConfig.withRegexAllowList = len(handler.forwardQueryParamsConfig.regexAllowList) > 0
 		}
-		handler.handlerPool = pond.New(
```

Comment:
@jensneuse Not sure, why we have created a worker pool per websocket request. It seems overkill.

Reply:
Actually I think we should move the complete handler init up.
We're also creating the ePoll instance within the middleware loop which seems odd.

Reply:
Great observation. Fixed.